client/src/components/DatasetInformation/DatasetInformation.vue

```diff
@@ -100,7 +100,10 @@ export default {
     },
     methods: {
         bytesToString(raw_size) {
-            return bytesToString(raw_size, false);
+            return Utils.bytesToString(raw_size, false);
```

Comment:
Why are we changing this here? Explicitly importing the individual function seems to be a better practice.

Reply:
If I remember well, it was the linter suggesting this change (probably from my VSCode). I would not have changed it myself. I will revert it back!

Reply:
I forgot to mention that I extracted this non-viz bug into https://github.com/galaxyproject/galaxy/pull/16301. I added this suggestion there too.